spec/02-requests/05-presign_v4_spec.lua

```diff
@@ -0,0 +1,76 @@
+setmetatable(_G, nil)
+
+-- to get a definitive result
+-- luacheck:ignore
+ngx.time = function()
+  return 1667543171
+end
```

Comment:
This changes the global environment, and hence will spill over to other tests.

This should probably be in a `setup` block, with a matching `teardown` block to undo the change

Reply:
fixed